wrapanapi/rest_client.py

```diff
@@ -2,10 +2,13 @@
 import os
 import json
 import six
+import logging
 
 from exceptions import RestClientException
 
 requests.packages.urllib3.disable_warnings()
+logging.basicConfig(level=logging.INFO)
```

Comment:
I would prefer to see the logger setup within the ContainerClient constructor as an object attribute, the same way that `WrapanapiAPIBase` does:
https://github.com/ManageIQ/wrapanapi/blob/master/wrapanapi/base.py#L37

Then access the logger through `self.logger` in the instance methods - everywhere you've added logging takes `self`.